src/components/organisms/AppRouter/AdminSubrouter.tsx

```diff
@@ -5,31 +5,100 @@ import Admin from "pages/Admin/Admin";
 import { RoomsForm } from "pages/Admin/Venue/Rooms/RoomsForm";
 import { VenueWizard } from "pages/Admin/Venue/VenueWizard";
 
-import Admin_v2 from "pages/Admin/Admin_v2";
-import VenueWizard_v2 from "pages/Admin/Venue/VenueWizard/VenueWizard";
+import AdminV2 from "pages/Admin/Admin_v2";
+import VenueWizardV2 from "pages/Admin/Venue/VenueWizard/VenueWizard";
```

Comment:
Why did we rename these imports here?

Reply:
There was browser console wrnings about having `_` underscore in the name.
So to reduce that chatter and since they're already named locally to `AdminSubrouter.tsx` anyway, 
I just removed that one char to make them camelCase

Reply:
> There was browser console wrnings about having _ underscore in the name.

I've literally never seen that anywhere. Can you please reproduce and then provide screenshot/further details.

Reply:
Strange.. I wonder if this is something new, or if i've just never noticed it before:

![image](https://user-images.githubusercontent.com/753891/125151585-7cc5b400-e18a-11eb-9e74-fa2f98313ee8.png)

Either way, change seems fine.